sql_generators/derived_view_schemas/__init__.py

```diff
@@ -11,12 +11,14 @@
     "_external",
     "_bi",
     "_restricted",
+    "udf",
```

Comment:
does `udf_js` also need to be added here?

Reply:
The way the check is written: https://github.com/mozilla/bigquery-etl/blob/9677405044571c40b3490861eacbcc9d86d6d917/sql_generators/derived_view_schemas/__init__.py#L166-L169 should cover the `udf` substring in `udf_js`, `udf_js_ilb` and `udf_legacy` as well. (I guess that makes "suffix" a little misleading here haha, updated)